tutorials/mpi-fortran/code/reduce_std.f90

```diff
@@ -0,0 +1,89 @@
+module subs
+  implicit none
+contains
+  subroutine create_rand_nums(rand_nums, num_elements)
+    ! Creates an array of random numbers. Each number has a value from 0 - 1
+    integer, intent(in) :: num_elements
+    real, intent(out)   :: rand_nums(num_elements)
+
+    integer :: i
+
+    do i = 1, num_elements
+      rand_nums(i) = rand()
+    end do
+
+  end subroutine create_rand_nums
+end module subs
+
+program main
+  use mpi_f08
+  use iso_fortran_env, only: error_unit
+  use subs
+
+  implicit none
+
+  integer :: num_args
+  character(12) :: arg
```

Comment:
As per previous suggestion make the 12 a parameter than can be imported from a module?